.github/workflows/ci-workflow.yml

```diff
@@ -75,40 +75,15 @@ jobs:
       run: npm run services
     - name: Run Integration Tests
       run: npm run integration
-
-  secrets:
-    runs-on: ubuntu-latest
-
-    if: github.repository == 'newrelic/node-newrelic'
-
-    strategy:
-      matrix:
-        node-version: [8.x, 10.x, 12.x]
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Install Dependencies
-      run: npm ci
-    - name: Run Docker Services
-      run: npm run services
-    - name: Run Versioned Tests
-      run: npm run secrets
       env:
+        TEST_LICENSE: ${{ secrets.TEST_LICENSE }}
+        COLLECTOR_LICENSE: ${{ secrets.COLLECTOR_LICENSE }}
         LASP_LICENSE: ${{ secrets.LASP_LICENSE }}
         LASP_SECURE_LICENSE: ${{ secrets.LASP_SECURE_LICENSE }}
-        BENDER_LICENSE: ${{ secrets.BENDER_LICENSE }}
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
```

Comment:
I see we're removing the AWS_ACCESS_KEY_ID variables, which (based on other changes) means we won't run the AWS SDK versioned tests. Presuming this is intentional -- if not see-something/say-something.

Reply:
this is only impacting integration tests. the versioned tests still have the secrets.